### PR TITLE
OADP-3904: fix and update the velero commands in openshift doc for [d…

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-3904](https://issues.redhat.com/browse/OADP-3904)

### Version(s):

* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### Link to docs preview:

* [Enabling self-signed CA certificates](https://74658--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.html#oadp-self-signed-certificate_installing-oadp-ocs)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@weshayutin && @stillalearner - if you only require small changes, I can get this published in the next couple of days
Thanks

**Review by weshayutin — APPROVED** [review]
/LGTM

thank you!!! extra effort :)

File: modules/oadp-self-signed-certificate.adoc [hunk 1]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero backup describe  <backup-name> --details --cacert /tmp/your-cacert.txt
```
Comment (PrasadJoshi12) [inline modules/oadp-self-signed-certificate.adoc:84]: It should be velero describe backup.
```suggestion
$ velero  describe backup  <backup-name> --details --cacert /tmp/your-cacert.txt
```

**Comment by anarnold97** [comment]
@PrasadJoshi12

File: modules/oadp-self-signed-certificate.adoc [hunk 2]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero  describe backup  <backup-name> --details --cacert /tmp/your-cacert.txt
```
Comment (PrasadJoshi12) [inline modules/oadp-self-signed-certificate.adoc:84]: Just a small suggestion 
```suggestion
$ velero describe backup  <backup-name> -n <oadp-namespace>--details --cacert /tmp/your-cacert.txt
```

File: modules/oadp-self-signed-certificate.adoc [hunk 3]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
...
 ----
+
+. To fetch the logs for this specific backup, which are useful for viewing failures and warnings, including resources that could not be backed up, run the following command:
 +
+[source,terminal]
+----
+$ velero backup logs  <backup-name>  --cacert /tmp/your-cacert.txt
```
Comment (PrasadJoshi12) [inline modules/oadp-self-signed-certificate.adoc:91]:
```suggestion
$ velero backup logs  <backup-name> -n <oadp namespace> --cacert /tmp/your-cacert.txt
```

File: modules/oadp-self-signed-certificate.adoc [hunk 4]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
...
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero  describe backup  <backup-name> --details --cacert /tmp/your-cacert.txt
 ----
+
+. To fetch the logs for this specific backup, which are useful for viewing failures and warnings, including resources that could not be backed up, run the following command:
 +
+[source,terminal]
+----
+$ velero backup logs  <backup-name> -n <oadp namespace> --cacert /tmp/your-cacert.txt
```
Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc:91]:
```suggestion
$ velero backup logs  <backup-name>  --cacert /tmp/your-cacert.txt
```

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/oadp-self-signed-certificate.adoc (hunk 2 above)
Comment (xenolinux) [inline modules/oadp-self-signed-certificate.adoc:84]: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

> Separated by underscores (_) for multi-word values


```suggestion
$ velero  describe backup  <backup_name> --details --cacert /tmp/<your_cacert.txt>
```

File: modules/oadp-self-signed-certificate.adoc (hunk 3 above)
Comment (xenolinux) [inline modules/oadp-self-signed-certificate.adoc:91]:
```suggestion
$ velero backup logs  <backup_name>  --cacert /tmp/<your_cacert.txt>
```

File: modules/oadp-self-signed-certificate.adoc [hunk 5]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero  describe backup  <backup-name> --details --cacert /tmp/your-cacert.txt
 ----
+
+. To fetch the logs for this specific backup, which are useful for viewing failures and warnings, including resources that could not be backed up, run the following command:
```
Comment (xenolinux) [inline modules/oadp-self-signed-certificate.adoc:87]:
```suggestion
. You can use the logs to view failures and warnings for the resources that you cannot back up. To fetch the backup logs run the following command:
```

**Review by xenolinux — COMMENTED** [review]
Few comments; otherwise LGTM

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: modules/oadp-self-signed-certificate.adoc (hunk 2 above)
Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc:84]:
```suggestion
$ velero  describe backup  <backup_name> --details --cacert /tmp/<your_cacert>.txt
```

File: modules/oadp-self-signed-certificate.adoc [hunk 6]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero  describe backup  <backup_name> --details --cacert /tmp/<your_cacert>.txt
```
Comment (maxwelldb) [inline modules/oadp-self-signed-certificate.adoc:84]:
```suggestion
$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
```

Assuming that the double spacing is not semantic. ;)

File: modules/oadp-self-signed-certificate.adoc (hunk 3 above)
Comment (maxwelldb) [reply modules/oadp-self-signed-certificate.adoc:91]: @anarnold97 Something like this change still needs to be applied.

**Review by maxwelldb — COMMENTED** [review]
Left a couple of comments from merge review.

File: modules/oadp-self-signed-certificate.adoc [hunk 7]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
...
 ----
+
+. You can use the logs to view failures and warnings for the resources you cannot back up. To fetch the backup logs run the following command:
```
Comment (maxwelldb) [inline modules/oadp-self-signed-certificate.adoc:87]: Could structure this as:

```
.To fetch the backup logs, run the following command:
# ...
You can use these logs to view failures and warnings for the resources that you cannot back up.
```

File: modules/oadp-self-signed-certificate.adoc [hunk 8]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
 ----
+
+. You can use the logs to view failures and warnings for the resources you cannot back up. To fetch the backup logs run the following command:
```
Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc:87]:
```suggestion
.To fetch the backup logs, run the following command:
```

File: modules/oadp-self-signed-certificate.adoc [hunk 9]
```diff
@@ -80,9 +81,16 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
...
+$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
 ----
+
+.To fetch the backup logs, run the following command:
 +
+[source,terminal]
+----
+$ velero backup logs  <backup_name>  --cacert /tmp/<your_cacert.txt>
+----
+
```
Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc:93]:
```suggestion
+
You can use these logs to view failures and warnings for the resources that you cannot back up.

```

File: modules/oadp-self-signed-certificate.adoc [hunk 10]
```diff
@@ -80,9 +81,18 @@ $ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i
 +
 [source,terminal]
 ----
-$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
+$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
 ----
+
+.To fetch the backup logs, run the following command:
```
Comment (sheriff-rh) [inline modules/oadp-self-signed-certificate.adoc:87]: This needs a space to be treated as a step. It also renders incorrectly in your build preview. 
image

**Review by sheriff-rh — CHANGES_REQUESTED** [review]
I think this will render correctly in the build preview if you fix the step.

File: modules/oadp-self-signed-certificate.adoc (hunk 10 above)
Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc:87]:
```suggestion
. To fetch the backup logs, run the following command:
```

Comment (anarnold97) [reply modules/oadp-self-signed-certificate.adoc]: Updated and squashed

Comment (anarnold97) [inline modules/oadp-self-signed-certificate.adoc]:
```suggestion
. To fetch the backup logs, run the following command:
```

**Comment by weshayutin** [comment]
@anarnold97  We have instructions for using the velero CI in two spots w/ different instructions to some degree.

* Configuring OADP with ODF && troubleshooting.

```
To use a CA certificate with this command, you can add a certificate to the Velero deployment by running the following commands:


$ CA_CERT=$(oc -n openshift-adp get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}')

$ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt" || echo "DPA BSL has no caCert"
```
and
```
$ velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt

``
Is relevant in both sections.  Can we please have it available in both as you see fit to do it.
Ping if you have questions :)

File: modules/migration-debugging-velero-resources.adoc [hunk 11]
```diff
@@ -42,6 +42,8 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
   --help
 ----
 
+// Wes this is the describe section. What changes do I need to make
```
Comment (anarnold97) [inline modules/migration-debugging-velero-resources.adoc:45]: Wes this is the `describe` module in the Troubleshooting Guide

What do I need to change?

File: backup_and_restore/application_backup_and_restore/troubleshooting.adoc [hunk 12]
```diff
@@ -26,6 +26,7 @@ include::modules/velero-oadp-version-relationship.adoc[leveloffset=+2]
 include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
+// @Wes debugging the Velero resources
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/troubleshooting.adoc:29]: @weshayutin - i could include the oadp-self-signed-certificate offset=+2if that would help

**Comment by anarnold97** [comment]
@weshayutin - what I have changed in the `oadp-self-signed-certificate.adoc` module which is included in six files

/home/anarnold/Documents/SublimeMerge/openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc:
   33  
   34  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   35: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   36  
   37  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

/home/anarnold/Documents/SublimeMerge/openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc:
   30  
   31  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   32: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   33  
   34  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

/home/anarnold/Documents/SublimeMerge/openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc:
   30  
   31  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   32: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   33  
   34  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

/home/anarnold/Documents/SublimeMerge/openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc:
   36  
   37  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   38: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   39  
   40  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

/home/anarnold/Documents/SublimeMerge/openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc:
   40  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   41  include::modules/oadp-odf-cpu-memory-requirements.adoc[leveloffset=+3]
   42: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   43  
   44  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

/home/anarnold/Documents/SublimeMerge/openshift-docs/virt/backup_restore/virt-installing-configuring-oadp.adoc:
   26  
   27  include::modules/oadp-setting-resource-limits-and-requests.adoc[leveloffset=+2]
   28: include::modules/oadp-self-signed-certificate.adoc[leveloffset=+2]
   29  
   30  include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]

What do I need to change in the `migration-debugging-velero-resources.adoc`


I will need to get this re-signed off by QE as this is a substantive change

**Review by weshayutin — APPROVED** [review]
Let's go with what you have now, thanks @anarnold97 
/LGTM

File: modules/migration-debugging-velero-resources.adoc (hunk 11 above)
Comment (kcarmichael08) [reply modules/migration-debugging-velero-resources.adoc:45]: Did you want to remove these?

File: backup_and_restore/application_backup_and_restore/troubleshooting.adoc (hunk 12 above)
Comment (kcarmichael08) [inline backup_and_restore/application_backup_and_restore/troubleshooting.adoc:29]: remove?

**Review by kcarmichael08 — COMMENTED** [review]
Wondering if you wanted to remove the placeholder comments - everything else lgtm!

**Comment by kcarmichael08** [comment]
Verified that @sheriff-rh 's requested change was made.

File: backup_and_restore/application_backup_and_restore/troubleshooting.adoc (hunk 12 above)
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/troubleshooting.adoc:29]:
```suggestion
```

File: modules/migration-debugging-velero-resources.adoc (hunk 11 above)
Comment (anarnold97) [inline modules/migration-debugging-velero-resources.adoc:45]:
```suggestion
```

File: modules/oadp-self-signed-certificate.adoc
Comment (skopacz1) [inline modules/oadp-self-signed-certificate.adoc:1]: This is out of scope so I won't hold up merging over this, but this procedure needs a `.Procedure` title at the beginning of it, above the first step on line 53. Just something to keep in mind if you have the time to fix this in a separate PR.

**Review by skopacz1 — APPROVED** [review]
Left one out of scope comment that I won't hold up merging over - just something to keep in mind if you have the time to fix it later. Otherwise merge review LGTM, proceeding to merge now.

**Comment by skopacz1** [comment]
Cherry picks merged

**Pull request merged**